client/types/disputes.d.ts

```diff
@@ -0,0 +1,52 @@
+// eslint-disable-next-line wpcalypso/import-docblock
+import { Order } from '@woocommerce/api';
+import { Charge } from './charges';
+
+interface Evidence {
+	[ key: string ]:
+		| string
+		| Record< string, boolean >
+		| Record< string, string >;
+	isUploading: Record< string, boolean >;
+	metadata: Record< string, string >;
+	uploadingErrors: Record< string, string >;
+}
+
+interface EvidenceDetails {
+	has_evidence: boolean;
+	due_by: number;
+}
+
+export interface Dispute {
+	status: string;
+	id: string;
+	evidence_details?: EvidenceDetails;
+	metadata: Record< string, string >;
+	productType: string;
```

Comment:
This should not be part of the Dispute interface. 

If we look at https://github.com/Automattic/woocommerce-payments/blob/develop/client/disputes/evidence/index.js#L378 we'll see that productType for is taken from `dispute.metadata` object

```
let productType = dispute.metadata[ PRODUCT_TYPE_META_KEY ] || '';
```

Also, it's never used as property of dispute. 

This was taken from [a recent PR ](https://github.com/Automattic/woocommerce-payments/pull/2773/files#diff-340a2de153572380f61d1d4e3c43e10b6b07619748daa6dee1969fd9f1ee9d37R12) and I suspect that the mistake is already there. 

Checking against API response also affirms that `Dispute` doesn't have property `productType`.

Reply:
Good catch, corrected in 0d2db988c861e1fbdb600b32267ee9d9facd18bb